fleet/lib/rancher-backup/values.yaml

```diff
@@ -0,0 +1,18 @@
+## Default s3 bucket for storing all backup files created by the backup-restore-operator
+s3:
+  enabled: true
+  credentialSecretName: rancher-bkp
+  credentialSecretNamespace: cattle-resources-system
+  region: us-east-1
+  bucketName: rancher-bkp-${ .ClusterLabels.site }
+  folder: ${ .ClusterLabels.site }
+  endpoint: s3.us-east-1.amazonaws.com
+
+# Add log level flags to backup-restore
+debug: true
+trace: true
+
+global:
+  cattle:
+    psp:
+      enabled: false  # PSP enablement should default to false
```

Comment:
Surely, this defaults to false?

Reply:
i left it because i was trying to figure out what it was, now that i know that Pod Security is gone from 1.25, i can safely remove it